devimint/src/lib.rs

```diff
@@ -389,6 +441,20 @@ impl Gatewayd {
         })
     }
 
+    pub async fn stop_lightning_node(&mut self) -> Result<()> {
+        match &mut self.ln {
```

Comment:
I think `self.ln` should be removed, we still a new LnKind type for cmd bellow.

Reply:
I'm going to replace this with a trait, I think it will simplify some things

Reply:
Instead of making a new trait, I just saved the name of the Lightning Node that the gateway is currently connected to.